doc/classes/Performance.xml

```diff
@@ -131,67 +131,97 @@
 		</constant>
-		<constant name="RENDER_TEXTURE_MEM_USED" value="14" enum="Monitor">
+		<constant name="RENDER_TEXTURE_MEM_USED" value="15" enum="Monitor">
 			The amount of texture memory used (in bytes). [i]Lower is better.[/i]
 		</constant>
-		<constant name="RENDER_BUFFER_MEM_USED" value="15" enum="Monitor">
+		<constant name="RENDER_BUFFER_MEM_USED" value="16" enum="Monitor">
 			The amount of render buffer memory used (in bytes). [i]Lower is better.[/i]
 		</constant>
-		<constant name="PHYSICS_2D_ACTIVE_OBJECTS" value="16" enum="Monitor">
+		<constant name="PHYSICS_2D_ACTIVE_OBJECTS" value="17" enum="Monitor">
 			Number of active [RigidBody2D] nodes in the game. [i]Lower is better.[/i]
 		</constant>
-		<constant name="PHYSICS_2D_COLLISION_PAIRS" value="17" enum="Monitor">
+		<constant name="PHYSICS_2D_COLLISION_PAIRS" value="18" enum="Monitor">
 			Number of collision pairs in the 2D physics engine. [i]Lower is better.[/i]
 		</constant>
-		<constant name="PHYSICS_2D_ISLAND_COUNT" value="18" enum="Monitor">
+		<constant name="PHYSICS_2D_ISLAND_COUNT" value="19" enum="Monitor">
 			Number of islands in the 2D physics engine. [i]Lower is better.[/i]
 		</constant>
-		<constant name="PHYSICS_3D_ACTIVE_OBJECTS" value="19" enum="Monitor">
+		<constant name="PHYSICS_3D_ACTIVE_OBJECTS" value="20" enum="Monitor">
 			Number of active [RigidBody3D] and [VehicleBody3D] nodes in the game. [i]Lower is better.[/i]
 		</constant>
-		<constant name="PHYSICS_3D_COLLISION_PAIRS" value="20" enum="Monitor">
+		<constant name="PHYSICS_3D_COLLISION_PAIRS" value="21" enum="Monitor">
 			Number of collision pairs in the 3D physics engine. [i]Lower is better.[/i]
 		</constant>
-		<constant name="PHYSICS_3D_ISLAND_COUNT" value="21" enum="Monitor">
+		<constant name="PHYSICS_3D_ISLAND_COUNT" value="22" enum="Monitor">
 			Number of islands in the 3D physics engine. [i]Lower is better.[/i]
 		</constant>
-		<constant name="AUDIO_OUTPUT_LATENCY" value="22" enum="Monitor">
+		<constant name="AUDIO_OUTPUT_LATENCY" value="23" enum="Monitor">
 			Output latency of the [AudioServer]. [i]Lower is better.[/i]
 		</constant>
-		<constant name="MONITOR_MAX" value="23" enum="Monitor">
+		<constant name="NAVIGATION_ACTIVE_MAPS" value="24" enum="Monitor">
+			Number of active navigation maps in the [NavigationServer3D]. This also includes the two empty default navigation maps created by World2D and World3D. [i]Lower is better.[/i]
+		</constant>
+		<constant name="NAVIGATION_REGION_COUNT" value="25" enum="Monitor">
+			Number of active navigation regions in the [NavigationServer3D]. [i]Lower is better.[/i]
+		</constant>
+		<constant name="NAVIGATION_AGENT_COUNT" value="26" enum="Monitor">
+			Number of active navigation agents processing avoidance in the [NavigationServer3D]. [i]Lower is better.[/i]
```

Comment:
Should we drop the `Lower is better` hints?  I'm not sure if they actually apply to most of these metrics.

Reply:
I just copied from other monitors but since all those values are very relative how good / bad they are might be better to remove this hint.